src/NServiceBus.Metrics.PerformanceCounters/PerformanceCountersFeature.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Features;
+
+class PerformanceCountersFeature : Feature
+{
+    public PerformanceCountersFeature()
+    {
+        Defaults(s =>
+        {
+            options = s.EnableMetrics();
+            s.SetDefault(UpdateIntervalKey, TimeSpan.FromSeconds(2));
+        });
+    }
+
+    protected override void Setup(FeatureConfigurationContext context)
+    {
+        var logicalAddress = context.Settings.LogicalAddress();
+
+        var legacyInstanceNameMap = new Dictionary<string, CounterInstanceName?>
```

Comment:
@kbaley I talked this through with @gbiellem Removing the abbrevation and the random spacing would break all existing performance counters. This means all devops scripts and infrastructure that is built on top of these performance counters would need to change as well. We concluded it is not worth breaking it.

About JSON.NET. Not really. We use it to turn the payload into JObjects in the updater.